### PR TITLE
Improve the template for snapcraft init

### PR DESCRIPTION
**Pull request by sergiusens**
LP: #1575581

### CODE REVIEW

**Comment by kyrofa** [comment]
Meh.... I'm so-so on this. Running "`snapcraft` to get started" will certainly _not_ do what the developer wants. I like the additions of the placeholder data in the global metadata, but I'm not a huge fan of the default part since it'll never do anything useful. I'm :+1: on the code though if we think it's necessary.

**Comment by sergiusens** [comment]
@kyrofa imagine someone just getting started for the first time. Yeah it goes hand in hand with `snapcraft examples`

**Comment by elopio** [comment]
I like this a lot. Every time I run snapcraft init I have to open an existing snapcraft.yaml to see fill it. This will make it a lot easier.

File: integration_tests/test_snap.py [hunk 1]
```diff
@@ -147,3 +147,10 @@ def test_snap_with_exposed_files(self):
         self.assertThat(
             os.path.join(project_dir, 'snap', 'usr', 'bin', 'nmtui'),
             Not(FileExists()))
+
+    def test_snap_from_snapcraft_init(self):
+        self.assertThat('snapcraft.yaml', Not(FileExists()))
+        self.run_snapcraft('init')
+        self.assertThat('snapcraft.yaml', FileExists())
+
+        self.run_snapcraft('snap')
```
Comment (elopio) [inline integration_tests/test_snap.py]: awesome! :tada:

**Pull request merged**